lib/beaker/host/unix.rb

```diff
@@ -138,6 +138,23 @@ def external_copy_base
       @external_copy_base
     end
 
+    # Tells you whether a host platform supports beaker's
+    #   {Beaker::HostPrebuiltSteps#set_env} method
+    #
+    # @return [String,nil] Reason message if set_env is not supported,
+    #   nil if it is supported
+    def supports_set_env
```

Comment:
nit: It seems odd to me that a method `supports_set_env` returns `nil` to communicate that it does support that action and a truthy value to say that it doesn't support that action.